### DISCUSSION TITLE
Migrate from Discord to GitHub's Discussions

### DISCUSSION THREAD

**Discussion by igor-baiborodine** [Lednerb/bilberry-hugo-theme#314] [Ideas]
@Lednerb 

Hi Sascha,

Let me know what do you think of the following suggestions regarding how to organize and provide support:

1. Migrate all discussions from Discord to GitHub's Discussion. The main advantage here is to keep the project and related discussions in one place. If Discord is no longer there one day, we will not lose our discussion topics archive. 

2. With GitHub's Discussions, it should be simpler to organize and manage incoming requests. It has all the necessary discussions categories. We will keep only the real issues and bugs in the Issues. Some current items from the Issues section could be converted to discussion topics; for example, issue #[185](https://github.com/Lednerb/bilberry-hugo-theme/issues/185), could be converted into an Ideas discussion.

**Comment by Lednerb** [comment]
Hi @igor-baiborodine,

thanks for your great improvements and suggestions.

**Regarding 1:**
I also think that the Discord Channel is not the ideal place for discussions. When I've created the channel I hoped to have more time for moderation and to answer questions. Back in 2017 as this theme was published, GitHub Discussions did not exist so I've decided to create the Discord channel.

Switching to GitHub Discussions is the right decision.


**Regarding 2:**
This is the logical next step and should also be done.
I also don't like that feature requests and general ideas are kept as "issue". 
We should implement your suggestion directly.

**Comment by igor-baiborodine** [comment]
@Lednerb, following the migration from Discord to GitHub Discussions, I made some changes to the example site, namely the [Support and Discussions](https://github.com/Lednerb/bilberry-hugo-theme/blob/master/exampleSite/content/article/support-and-discussions/index.md) article. Then I translated it to German using Google Translate. So could you please validate the content of the [German version](https://github.com/Lednerb/bilberry-hugo-theme/blob/master/exampleSite/content/article/support-and-discussions/index.de.md)?

**Reply by Lednerb** [reply]
Thank you for your efforts @igor-baiborodine !

I will check the German version later this day.

**Reply by igor-baiborodine** [reply]
@Lednerb, here is another one to validate: https://github.com/Lednerb/bilberry-hugo-theme/blob/master/exampleSite/content/status/bilberry-theme-released.de.md

**Reply by Lednerb** [reply]
Please have a look at #323
